microcosm_pubsub/tests/test_registry.py

```diff
@@ -86,6 +92,18 @@ def test_find(self):
         assert_that(schema, is_(instance_of(PubSubMessageCodec)))
         assert_that(schema.schema, is_(instance_of(DerivedSchema)))
 
+    def test_find_wildcard(self):
+        schema = self.registry.find("application/vnd.microcosm.wildcard.foo_12")
+        assert_that(schema, is_(instance_of(PubSubMessageCodec)))
+        assert_that(schema.schema, is_(instance_of(WildcardSchema)))
+
+    def test_wildcard_does_not_match_extended(self):
+        # The postfix `*` wildcard doesn't match on `.` separators
+        assert_that(
+            calling(self.registry.find).with_args("application/vnd.microcosm.wildcard.foo.bar"),
+            raises(KeyError),
```

Comment:
Is this the `raise KeyError("Unregistered media type: {}".format(media_type))` case that is raised?

Reply:
Yeah that's the one.